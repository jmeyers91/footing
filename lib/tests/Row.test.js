"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var react_1 = __importDefault(require("react"));
var react_2 = require("@testing-library/react");
var Row_1 = __importDefault(require("../../src/Row"));
describe('Row', function () {
    test('Should match snapshot', function () {
        var container = react_2.render(react_1.default.createElement(Row_1.default, null)).container;
        expect(container.firstChild).toMatchSnapshot();
    });
});
//# sourceMappingURL=Row.test.js.map