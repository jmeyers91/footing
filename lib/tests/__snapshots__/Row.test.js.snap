// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Should match snapshot 1`] = `
<div
  class="sc-bdVaJa iCRaLp"
/>
`;
